### ISSUE TITLE
Feature Request: entitlements

### ISSUE THREAD

**Issue by getaaron** [CocoaPods/Xcodeproj#562]
An Xcode target can have certain 'capabilities' like iCloud, Wallet, Apple Pay, etc. Would love the ability to easily get programmatic access to a target's entitlements, e.g.:

```ruby
project.targets.first.enabled_entitlements
=> ["com.apple.developer.pass-type-identifiers"]

project.targets.first.enabled_entitlements.pretty_print
=> ["Wallet"]

project.targets.first.entitlements_files
=> ["SomeProject/SomeProject.entitlement"]
```

**Comment by keith** [comment]
FWIW the source of truth for these settings is the entitlements files for the targets. We don’t set this eve in our generates project yet they appear correctly

**Comment by getaaron** [comment]
@keith Yeah I mostly am interested for now in Xcodeproj parsing the entitlements files for me for reading purposes (see the linked fastlane issue). Although eventually it might be nice to manipulate them with Xcodeproj, for now I'm just looking for help parsing it.
